website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 layout: docs
-page_title: Ingress Controller Integrations
-description: Configuring Ingress Controllers With Consul On Kubernetes
+page_title: Configure Ingress Controllers for Consul on Kubernetes
+description: >-
+Ingress controllers are pluggable components that must be configured in k8s in order to use the Ingress resource. Learn how to deploy sidecars with the controller to secure its communication with Consul, review common configuration issues, and find links to example configurations. 
```

Comment:
```suggestion
Ingress controllers are pluggable components that must be configured in Kubernetes in order to use the Ingress resource. Learn how to deploy sidecars with the controller to secure its communication with Consul, review common configuration issues, and find links to example configurations. 
```
Did you use 'K8s' for length reasons or to target an alternate SEO term? This suggestion is just to flag this instance. Feel free to ignore.